3506 - Changes to TDP Home and KC following audit

reitermb opened the pull request:
## Summary of Changes
Closes #3506 

## How to Test
Screenshots and alt text updated on all pages where we have navigation/heading/interface changes. 
Test Environment: https://reitermb.github.io/Test/knowledge-center/index.html

## Accessible
Accessibility coverage for knowledge center is unaffected as the only changes were switching out screenshots. Alt text was retained as it was. 

TDP Home a11y scan: 
![image](https://github.com/user-attachments/assets/ecc301de-2ab5-4028-b984-680e89540e39)

Preview of app home change (removal of deadline table): 
![image](https://github.com/user-attachments/assets/9e8d4b45-34d7-4b66-9a34-7bff8b1cbd6b)

reitermb committed:
Commits all changes

victoriaatraft reviewed (changes requested):
Looking good @reitermb! Few minor things I noticed:

- [x] The government banner is missing from the home page screenshots
- [x] [Create new login page](https://reitermb.github.io/Test/knowledge-center/create-new-login.html) is missing the ACF logo at the top, it's shown on https://secure.login.gov/ it's (it does show the placeholder on logging in on test sites)
- [x] [Submitting Data Files Tab ](https://reitermb.github.io/Test/knowledge-center/uploading-data.html)needs to be replaced with the new header and the content needs to reflect that ie: TANF Data Files and FRA Data Files
    - [x] How to submit data should also update the screenshot with the current header, fields and calendar table and body content may need to also reflect that. 
- [x] [How to view submission history Step 1](https://reitermb.github.io/Test/knowledge-center/view-submission-history.) should reflect the current Data Files header and fields
- [x] For [Using an Existing account ](https://reitermb.github.io/Test/knowledge-center/existing-login.html), if they have an ACF email they'll see that they can request access to regions. Don't know if we want to add that in there. 
- [x] [Managing Your Account tab](https://reitermb.github.io/Test/knowledge-center/managing-your-account.html), needs to update with the new header

Also to note, the TANF Data Files Page says `Fiscal Year*`while the FRA Data Files says `Fiscal Year (October - September)*` as the input fields, I think they may need to update to be consistent @elipe17 

cc @lhuxraft

reitermb commented:
> Looking good @reitermb! Few minor things I noticed:
> 
>     * [x]  The government banner is missing from the home page screenshots
> 
>     * [x]  [Create new login page](https://reitermb.github.io/Test/knowledge-center/create-new-login.html) is missing the ACF logo at the top, it's shown on https://secure.login.gov/ it's (it does show the placeholder on logging in on test sites)
> 
>     * [x]  [Submitting Data Files Tab ](https://reitermb.github.io/Test/knowledge-center/uploading-data.html)needs to be replaced with the new header and the content needs to reflect that ie: TANF Data Files and FRA Data Files
>       
>       * [x]  How to submit data should also update the screenshot with the current header, fields and calendar table and body content may need to also reflect that.
> 
>     * [x]  [How to view submission history Step 1](https://reitermb.github.io/Test/knowledge-center/view-submission-history.) should reflect the current Data Files header and fields
> 
>     * [x]  For [Using an Existing account ](https://reitermb.github.io/Test/knowledge-center/existing-login.html), if they have an ACF email they'll see that they can request access to regions. Don't know if we want to add that in there.
> 
>     * [x]  [Managing Your Account tab](https://reitermb.github.io/Test/knowledge-center/managing-your-account.html), needs to update with the new header
> 
> 
> Also to note, the TANF Data Files Page says `Fiscal Year*`while the FRA Data Files says `Fiscal Year (October - September)*` as the input fields, I think they may need to update to be consistent @elipe17
> 
> cc @lhuxraft

Good catches! 

I went for cropping on the login.gov screens to keep focus more on the relevant parts of the flow and hopefully reduce our screenshot dependencies in case of future changes to how those screens look. I'm keeping FRA visibility for the upcoming knowledge center work that will add those pages, and I think we should keep regional staff relevant content out of the main knowledge center. Everything else should be addressed now!

reitermb committed:
Test env parity with additions / release notes / review tweaks

reitermb committed:
Merge branch 'develop' into 3506-Kc-and-homepage

lhuxraft reviewed (changes requested):
Thanks for making these changes! Only a couple of things I saw missing:
1. The first screenshot of the 'Uploading Data' page hasn't been updated
2. It doesn't look like the file extension guide and/or screenshots have been updated in the test environment?

reitermb commented:
> Thanks for making these changes! Only a couple of things I saw missing:
> 
>     1. The first screenshot of the 'Uploading Data' page hasn't been updated
> 
>     2. It doesn't look like the file extension guide and/or screenshots have been updated in the test environment?

![1-tab-emphasis](https://github.com/user-attachments/assets/e7e98e0d-75da-492b-b9c0-8458ddcad830)
Is this the screenshot you're seeing on the uploading data page? That should be up to date! I'm purposefully keeping FRA separated to come later in FRA-dedicated content. 

And I haven't touched the file extension guide but I don't believe we have anything to change there. Let me know if I'm missing something!

lhuxraft commented:
> > Thanks for making these changes! Only a couple of things I saw missing:
> > ```
> > 1. The first screenshot of the 'Uploading Data' page hasn't been updated
> > 
> > 2. It doesn't look like the file extension guide and/or screenshots have been updated in the test environment?
> > ```
> 
> ![1-tab-emphasis](https://private-user-images.githubusercontent.com/28515874/422984309-e7e98e0d-75da-492b-b9c0-8458ddcad830.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE5OTUzNjksIm5iZiI6MTc0MTk5NTA2OSwicGF0aCI6Ii8yODUxNTg3NC80MjI5ODQzMDktZTdlOThlMGQtNzVkYS00OTJiLWI5YzAtODQ1OGRkY2FkODMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE0VDIzMzEwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhM2NjODdkNDIzNzI0MjQ5MzFjN2U5Yzg3Mjk4YTY3MThlMDg0MDk3ZjBhZWRkMjE2ZTdmOGI3YmQ4MjRmZWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.e_Wkap8g3rnwAvYpnU1ooLdy1Ue9EhkajQFFbLQWU2k) Is this the screenshot you're seeing on the uploading data page? That should be up to date! I'm purposefully keeping FRA separated to come later in FRA-dedicated content.
> 
> And I haven't touched the file extension guide but I don't believe we have anything to change there. Let me know if I'm missing something!

Oh cool, thanks for clarifying! The audit document mentioned that we wanted to make some changes to the 'File Extension Guide' page - I assume because the file extensions accepted for FRA will be different than TANF Also totally fine if we're expecting that work to happen as part of the other ticket.

ADPennington commented on product-updates/img/getting-started/existing-login-approved-tdp-home.png:
@reitermb @victoriaatraft @lhuxraft do we need to add any language to clarify that FRA data files tabs is only visible for STTs that are required (and approved) to submit FRA reports?

lhuxraft replied:
@reitermb I think maybe we add this to the KC somewhere but not on the homepage?

lhuxraft replied:
@reitermb to take a look at where he would like this to live

lhuxraft replied:
Suggestion: keep this PR suggested to everything non-FRA, additions to FRA will be handled in another ticket

ADPennington replied:
@ttran-hub see below 🔽

ADPennington replied:
we should also remove any evidence of admin and grafana tabs. @lhuxraft

ADPennington commented on product-updates/knowledge-center/complete-resubmissions.html:
```diff
@@ -294,7 +294,7 @@ <h1>Resubmitting Complete Data Files</h1>
         </p>
 <br>
-        <p>The new TANF Data Portal will not change reporting requirements related to which data needs to be reported. However, it will require a process adjustment to how some grantees currently submit revisions of their data—specifically that revisions to data already submitted to TDP for a given quarter will need to be submitted in full (i.e. complete data) rather than as partial data only con ...
+        <p>The new TANF Data Portal will not change reporting requirements related to which data needs to be reported. However, it will require a process adjustment to how some STTs (States, Tribes, and Territories) currently submit revisions of their data—specifically that revisions to data already submitted to TDP for a given quarter will need to be submitted in full (i.e. complete data) rather ...
```
@reitermb we decided on 3/20 to hold off on replacing `grantee` terminology for now. cc: @klinkoberstar

ADPennington commented on product-updates/knowledge-center/exporting-complete-data-using-ftanf.html:
```diff
@@ -298,7 +298,7 @@ <h1>Exporting Complete Data Using FTANF</h1>
       <h2>What is FTANF.exe?</h2>
 <br>
-      <p>FTANF.exe—sometimes known just as ‘FTANF’, ‘pTANF’, or ‘the executables’—is a legacy desktop application tool that helps TANF grantees prepare transmission files for all sections of their quarterly data report. Data can be imported from other systems or entered manually and – once in the tool – could be exported into the transmission-ready flatfile (.txt) format. Though FTANF is no longe ...
+      <p>FTANF.exe—sometimes known just as ‘FTANF’, ‘pTANF’, or ‘the executables’—is a legacy desktop application tool that helps TANF STTs (States, Tribes, and Territories) prepare transmission files for all sections of their quarterly data report. Data can be imported from other systems or entered manually and – once in the tool – could be exported into the transmission-ready flatfile (.txt) fo ...
```
same as above re: `grantees`

ADPennington commented on product-updates/knowledge-center/index.html:
```diff
@@ -279,15 +279,15 @@ <h1>Getting Started</h1>
         
         <p class="usa-intro">
-          The TANF Data Portal (TDP) is a new, secure, web-based data reporting system designed to improve the federal reporting experience for TANF grantees and federal staff.
+          The TANF Data Portal (TDP) is a new, secure, web-based data reporting system designed to improve the federal reporting experience for TANF STTs (States, Tribes, and Territories) and federal staff.
```
same as above re: `grantees`

ADPennington commented on product-updates/knowledge-center/index.html:
```diff
@@ -279,15 +279,15 @@ <h1>Getting Started</h1>
 
           <div style="min-height: 22px;"></div>
-        <p>TDP will allow grantees to easily submit accurate data and be confident that they have fulfilled their reporting requirements. This will reduce the burden on all users, improve data quality, inform better policy and programs, and ultimately help low-income families.
+        <p>TDP will allow STTs to easily submit accurate data and be confident that they have fulfilled their reporting requirements. This will reduce the burden on all users, improve data quality, inform better policy and programs, and ultimately help low-income families.
```
same as above re: `grantees`

ADPennington commented on product-updates/knowledge-center/uploading-data.html:
```diff
@@ -316,10 +316,10 @@ <h3 class="usa-process-list__heading">Choose which data you’ll be submitting</
+      <img src="../img/data-files/3-upload-buckets.png" alt="Upload components for Fiscal Year 2021 — Quarter 1 (October-December) including 'buckets' for Active, Closed, Aggregate, and Stratum sections">
         </a>
-        <p>Note: Section 4 is only required for grantees who have elected to submit a stratified sample of families served in their jurisdiction rather than data for all families.</p><br>
+        <p>Note: Section 4 is only required for STTs (States, Tribes, and Territories) who have elected to submit a stratified sample of families served in their jurisdiction rather than data for all families.</p><br>
```
same as above re: `grantees`

ADPennington commented on product-updates/knowledge-center/uploading-data.html:
```diff
@@ -334,7 +334,7 @@ <h3 class="usa-process-list__heading">Submit your files</h3>
       <img src="../img/getting-started/data-submitted.png" alt="Email example of the successfully submitted data files by section">
       </a>
-      <p>In the future, TDP will provide more immediate reports about your transmitted data. For now, please expect to receive the transmission reports you’re used to receiving via email within 48 hours of data submission. Note that Tribal TANF grantees do not receive transmission reports by email.</p>
+      <p>In the future, TDP will provide more immediate reports about your transmitted data. For now, please expect to receive the transmission reports you’re used to receiving via email within 48 hours of data submission. Note that Tribal TANF STTs do not receive transmission reports by email.</p>
```
same as above re: `grantees`

ADPennington reviewed (changes requested):
couple of changes requested and a question.

reitermb committed:
undo grantee change

reitermb committed:
Merge branch '3506-Kc-and-homepage' of https://github.com/raft-tech/TANF-app into 3506-Kc-and-homepage

reitermb committed:
Update index.html

reitermb committed:
Merge branch 'develop' into 3506-Kc-and-homepage

reitermb commented:
Switched "STT" instances back to "grantee", updated the Test environment accordingly, and merged in develop—should be good to go!

cc @ADPennington @klinkoberstar @lhuxraft

reitermb committed:
Merge branch 'develop' into 3506-Kc-and-homepage

reitermb committed:
Merge branch 'develop' into 3506-Kc-and-homepage

ADPennington commented:
@reitermb @lhuxraft  are we keeping this? https://github.com/raft-tech/TANF-app/blob/9dc9d5fb1842ed67aaa416cc0938abe976e8d9d8/product-updates/img/getting-started/existing-login-approved-tdp-home.png

lhuxraft commented:
> @reitermb @lhuxraft are we keeping this? https://github.com/raft-tech/TANF-app/blob/9dc9d5fb1842ed67aaa416cc0938abe976e8d9d8/product-updates/img/getting-started/existing-login-approved-tdp-home.png

Are you referencing [this comment](https://github.com/raft-tech/TANF-app/pull/3543#discussion_r2007849420)? As noted in the comment thread, FRA-specific changes to this page will be handled in #3508

ADPennington commented:
> > @reitermb @lhuxraft are we keeping this? https://github.com/raft-tech/TANF-app/blob/9dc9d5fb1842ed67aaa416cc0938abe976e8d9d8/product-updates/img/getting-started/existing-login-approved-tdp-home.png
> 
> Are you referencing [this comment](https://github.com/raft-tech/TANF-app/pull/3543#discussion_r2007849420)? As noted in the comment thread, FRA-specific changes to this page will be handled in #3508

i may have misunderstood the decision.  To confirm: are you saying that we are leaving FRA-related content in this PR  and pushing any changes to FRA content as part of the newer ticket? @lhuxraft 

update: accidentally pressed the wrong button.

lhuxraft commented:
> > > @reitermb @lhuxraft are we keeping this? https://github.com/raft-tech/TANF-app/blob/9dc9d5fb1842ed67aaa416cc0938abe976e8d9d8/product-updates/img/getting-started/existing-login-approved-tdp-home.png
> > 
> > 
> > Are you referencing [this comment](https://github.com/raft-tech/TANF-app/pull/3543#discussion_r2007849420)? As noted in the comment thread, FRA-specific changes to this page will be handled in #3508
> 
> i may have misunderstood the decision. To confirm: are you saying that we are leaving FRA-related content in this PR and pushing any changes to FRA content as part of the newer ticket? @lhuxraft
> 
> update: accidentally pressed the wrong button.

Yes! @reitermb will be making the suggested change above as part of #3508

ADPennington reviewed (approved):
👍🏾

reitermb committed:
Merge branch 'develop' into 3506-Kc-and-homepage

victoriaatraft reviewed (approved):
LGTM
